code/modules/events/event_dynamic.dm

```diff
@@ -20,7 +20,7 @@ var/list/event_last_fired = list()
 	if(universe.name != "Normal")
 		message_admins("Universe isn't normal, aborting random event spawn.")
 		return
-	if(player_list.len < 6) //minimum pop of 5 to trigger events
+	if(player_list.len < 1) //minimum pop of 1 to trigger events
```

Comment:
Check that logic.  You want a `<= 1` or this will NOT trigger on a player_list.len of `1`.

Reply:
Wait no this is fine. The old comment confused me.

Reply:
Thank you for the heads up.

Reply:
The addition of that `=` means that events will no longer fire with exactly 1 player. You should either choose `<1` or `==0`

Reply:
Oh, I see, I'll change it back to <1. That's what I intended to have events for lone players.